released/packages/coq-reduction-effects/coq-reduction-effects.0.1.3/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "Yishuai Li <yishuai@cis.upenn.edu>"
+
+homepage: "https://github.com/coq-community/reduction-effects"
+dev-repo: "git+https://github.com/coq-community/reduction-effects.git"
+bug-reports: "https://github.com/coq-community/reduction-effects/issues"
+license: "MPL-2.0"
+
+synopsis: "A Coq plugin to add reduction side effects to some Coq reduction strategies"
+
+build:   [make "-j%{jobs}%"]
+install: [make "-j%{jobs}%" "install"]
```

Comment:
```suggestion
install: [make "install"]
```
Is there an obvious reason multiple jobs are needed here? Just curious.

Reply:
Indeed no parallelism is needed.